server/src/models/Character.js

```diff
@@ -27,6 +29,24 @@ class Character extends Model {
           to: "hunters.index",
         },
       },
+
+      stats: {
+        relation: Model.HasOneRelation,
+        modelClass: Stats,
+        join: {
+          from: "characters.id",
+          to: "stats.characterId",
+        },
+      },
+
+      look: {
+        relation: Model.HasOneRelation,
+        modelClass: Look,
+        join: {
+          from: "characters.id",
+          to: "looks.characterId",
+        },
+      },
```

Comment:
generally, a hasOne relation can be simplified by literally just storing all of this look info ON the character table. It would be a lot of columns, but technically you could store all of this data on the character table. Up to you, there might be some reason you are doing this

Reply:
So... I have them on different tables because I need to create the character first (so I know what the hunter type is) and THEN grab these materials. Since stats are not nullable, I thought I would have to do a different stats table elsewhere.